### ISSUE TITLE
Cannot use text-track-display.js to display descriptions track at the same time as supporting Captions and Subtitles natively

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#4180]
## Description
On all browsers except Safari, video.js displays emulated text tracks via `text-track-display.js`. On Safari (macOS and iOS), video.js allows the browser to display captions and subtitles, unless `nativeTextTracks` (or `nativeCaptions`) is set to false.

However, Safari does not support text tracks with `kind="descriptions"` (they are just ignored) - setting `nativeTextTracks` to false means that Safari on macOS displays captions, subtitles and descriptions using `text-track-display.js`, but on iOS no subtitles, captions or descriptions are displayed. So the existing options do not allow a user to fix this problem unless they set `nativeTextTracks` based on detecting the browser and OS.

Until more browsers support `descriptions` tracks, and there's an effective way to detect that support, descriptions tracks should _always_ be displayed with `text-track-display.js` (although we could add another option `nativeTextDescriptionsTracks` to allow users to override this).

Note that we already have logic to prevent a descriptions track from being enabled (via the menu button) when a captions or subtitles track is enabled, and we also have logic which prevents descriptions from being displayed if a subtitles or captions track is showing, so we won't have a case where native captions visually overlap emulated descriptions.
